backend/app/user_handler.go

```diff
@@ -305,7 +308,7 @@ func (h *Handler) VerifyRegisterCode(c *gin.Context) {
 			return
 		}
 
-		h.sseManager.Notify(fmt.Sprintf("%d", user.ID), "user_registration", "User email is verified")
+		h.sseManager.Notify(fmt.Sprintf("%d", user.ID), "user_registration", models.NotificationSeverityInfo, map[string]string{"status": "User email is verified"})
```

Comment:
is that supposed to use sse directly like that or should go through the notification service?